server/src/main/java/io/deephaven/server/runner/updategraph/UpdateGraphModule.java

```diff
@@ -16,9 +19,20 @@ public class UpdateGraphModule {
     @Provides
     @Singleton
     @Named(PeriodicUpdateGraph.DEFAULT_UPDATE_GRAPH_NAME)
-    public static UpdateGraph provideUpdateGraph() {
+    public static UpdateGraph provideUpdateGraph(
+            final ThreadInitializationFactory threadInitializationFactory,
+            final OperationInitializer operationInitializer) {
         return PeriodicUpdateGraph.newBuilder(PeriodicUpdateGraph.DEFAULT_UPDATE_GRAPH_NAME)
                 .numUpdateThreads(PeriodicUpdateGraph.NUM_THREADS_DEFAULT_UPDATE_GRAPH)
+                .threadInitializationFactory(threadInitializationFactory)
```

Comment:
@niloc132 Was Python debugging broken without this?

Reply:
Likely, yes, but we don't have tests for this.